### PR TITLE
docs: Fix markdown

### LINKED ISSUE #433: Fix markdown
Resolve issues in the markdown files in this repo.

### PR DESCRIPTION
**Pull request by jodh-intel**
Resolve issues in the markdown files in this repo.

Fixes #433.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
Ugh. Anyone know what this URL needs to change to?

```
INFO: Checking all document URLs

ERROR: Invalid URL 'https://run.conduit.io/install' found in the following files:

svc-mesh/service-mesh.md
```

**Comment by grahamwhaley** [comment]
@jodh-intel - looks like it changed name - I suspect maybe https://linkerd.io/2/getting-started/ but /cc @amshinde @mcastelino

**Comment by amshinde** [comment]
@grahamwhaley correct. Looks like it got merged into linkerd project. So the url should now be "https://run.linkerd.io/install". We also need to rename all instances of conduit in that doc to linkerd.

**Comment by jodh-intel** [comment]
Thanks @amshinde - doc updated. Note that the https://raw.githubusercontent.com/runconduit/conduit-examples/master/emojivoto/emojivoto.yml URL is still valid and there appears to be no linkerd equivalent.

Since I've only done a search'n'replace, I'd appreciate it if a conduit/linkerd person could sanity-check this!

**Comment by jodh-intel** [comment]
Ping y'all!

/test

File: Limitations.md [hunk 1]
```diff
@@ -18,12 +17,12 @@
...
     * [Networking limitations](#networking-limitations)
         * [Support for joining an existing VM network](#support-for-joining-an-existing-vm-network)
-        * [docker --net=host](#docker---net=host)
+        * [docker --net=host](#docker---nethost)
         * [docker run --link](#docker-run---link)
     * [Host resource sharing](#host-resource-sharing)
         * [docker run --privileged](#docker-run---privileged)
 * [Miscellaneous](#miscellaneous)
-    * [Docker --security-opt partially supported](#docker---security-opt-option-partially-supported)
+    * [Docker --security-opt option partially supported](#docker---security-opt-option-partially-supported)
```
Comment (chavafg) [inline Limitations.md]: remove spaces at the end?

File: design/architecture.md [hunk 2]
```diff
@@ -630,7 +631,7 @@ with a Kubernetes pod:
 
 #### Mixing VM based and namespace based runtimes
 
-> **Note:** Since Kubernetes 1.12, the `[Kubernetes RuntimeClass](how-to/containerd-kata.md#kubernetes-runtimeclass)` 
+> **Note:** Since Kubernetes 1.12, the `[Kubernetes RuntimeClass](/how-to/containerd-kata.md#kubernetes-runtimeclass)`
```
Comment (chavafg) [inline design/architecture.md]: should this create a link?

File: Limitations.md (hunk 1 above)
Comment (jodh-intel) [reply Limitations.md]: Fixed.

File: design/architecture.md (hunk 2 above)
Comment (jodh-intel) [reply design/architecture.md]: Fixed.

**Comment by jodh-intel** [comment]
Good catches @chavafg! :smile:

**Comment by jodh-intel** [comment]
Well, I'm happy that we land this asap as it's better than what we have currently! :smile:

But I can't ack as it's my PR :)

Restarted Travis which seems to have had a "wobble" as the error is incorrect fwics:

```
INFO: Checking all document URLs
ERROR: Invalid URL 'https://api.github.com/repos/intel/ovmf-virt/releases/latest' found in the following files:
how-to/how-to-use-kata-containers-with-nemu.md
```

**Comment by jodh-intel** [comment]
Travis has now been disabused.

**Comment by jodh-intel** [comment]
Rocking - thanks @chavafg!

**Pull request merged**